static/images/Image-Licensing-and-Technical-Notes.txt

```diff
@@ -0,0 +1,77 @@
+Licensed under the SIL Open Font License, Version 1.1. See \nhttp://scripts.sil.org/OFL
+
+The license terms above apply to the following images:
+          FILENAME         ORIGINAL GLYPH     ORIGINAL FONT FILE
+    =====================  ==============     ==================
+    doubleleftarrow.svg       U+21D0             KaTeX Main
+    doubleleftrightarrow.svg  U+21D4             KaTeX Main
+    doublerightarrow.svg      U+21D2             KaTeX Main
+    hookleftarrow.svg         U+21A9             KaTeX Main
+    hookrightarrow.svg        U+21AA             KaTeX Main
+    leftarrow.svg             U+21A2             KaTeX AMS Regular
+    leftharpoon.svg           U+21BF             KaTeX AMS Regular
+    leftrightarrow.svg        U+21A2/21A3        KaTeX AMS Regular
+    leftrightharpoons.svg     U+21BC/21B1        KaTeX Main
+    mapsto.svg                U+21A6             KaTeX Main
+    overbrace.svg             U+23A9/23A8/23A7   KaTeX Size 4 Regular
+    rightarrow.svg            U+21A3             KaTeX AMS Regular
+    rightharpoon.svg          U+21C0             KaTeX AMS Regular
+    rightharpoondown.svg      U+21C1             KaTeX Main
+    rightleftharpoons.svg     U+21CC             KaTeX Main
+    tofrom.svg                U+21C4             KaTeX AMS Regular
+    twoheadleftarrow.svg      U+219E             KaTeX AMS Regular
+    twoheadrightarrow.svg     U+21A0             KaTeX AMS Regular
+    underbrace.svg            U+23A9/23A8/23A7   KaTeX Size 4 Regular
+    xleftarrow.svg            U+2190             KaTeX Main
+    xleftharpoon.svg          U+21BC             KaTeX Main
+    xleftharpoondown.svg      U+21BD             KaTeX Main
+    xleftrightarrow.svg       U+2194             KaTeX Main
+    xrightarrow.svg           U+2192             KaTeX Main
+    xrightharpoon.svg         U+21C0             KaTeX Main
+    xrightharpoondown.svg     U+21C1             KaTeX Main
+
+Images for \widehat, \widetilde, \overgroup, and \undergroup have been adapted (and modified)
+from font glyphs in the MnSymbol package. These fonts are in the public domain.
+
+
+Technical Comments
+==================
+
+Nested SVGs
+    Many of the KaTeX SVG images contain a nested SVG. This is done to achieve a
+    stretchy image while avoiding distortion of arrowheads or brace corners.
+
+    The inner SVG typically contains a very long (400 em) arrow.
+
+    The outer SVG acts like a window that exposes only part of the inner SVG. The
+    outer SVG will grow or shrink to match the dimensions set for it by CSS.
+
+    The inner SVG always has a longer, thinner aspect ratio than the outer SVG.
+    After the inner SVG fills 100% of the height of the outer SVG, there is a long
+    arrow shaft left over. That left-over shaft is not shown. Instead, it is
+    sliced off because the inner SVG is set to "preserveAspectRatio='... slice'".
+
+    Thus, the reader sees an arrow that matches the subject matter width without
+    distortion.
```

Comment:
The SVG spec also has a clever way to use nested SVGs that stretch to arbitrary length. In my research, I found bug reports to Microsoft (in 2015!) that complained that the stretchers were not working. So I started long and used a slice. Per my BrowserStack testing, this method works everywhere including IE 9+